change/@itwin-tree-widget-react-c5e52e86-ebe6-439d-a0c6-1a895dd61a3a.json

```diff
@@ -0,0 +1,7 @@
+{
+  "type": "patch",
+  "comment": "Node visibility button will no longer generate an empty tooltip, when tooltip content is empty.",
```

Comment:
Can you update change message to format: `Fixed: <what was fixed>` to highlight that this was a bug fix?

Reply:
changed